proto/content_analysis/sdk/analysis.proto

```diff
@@ -18,6 +18,15 @@ enum AnalysisConnector {
   PRINT = 4;
 }
 
+message Handshake {
+  enum Status {
+    STATUS_UNKNOWN = 0;
+    SUCCESS = 1;
+    FAILURE = 2;
+  }
+  required Status status = 1;
```

Comment:
What do you think of adding an optional process id of sender as a field of Handshake?  At least in the chrome->agent direction I think this could be quite useful.

Reply:
sg!